src/components/Ellipse/Ellipse.stories.tsx

```diff
@@ -5,12 +5,21 @@ import { Story } from "@storybook/react";
 import Ellipse, {EllipseProps} from "./Ellipse";
 
 export default {
-    title:"Icons/Ellipse",
+    title:"Components/Ellipse",
     component:Ellipse
  } as Meta
 
 const Template: Story<EllipseProps>=(args)=><Ellipse {...args} />
 export const Default = Template.bind({})
 Default.args={
     color:"hsla(278, 30%, 48%, 1)"
+    
+}
+export const WithAllProps = Template.bind({})
+
+WithAllProps.args={
+    color:"hsla(278, 30%, 48%, 1)",
```

Comment:
Please see about, this should be `status` instead of `color`. We do not want to embed rgbs/hsl codes into our code. It will break rebranding if we want.

Reply:
I see. Now status is what it is defined in app.variables.scss